backends/ebpf/psa/externs/ebpfPsaMeter.cpp

```diff
@@ -280,6 +268,62 @@ cstring EBPFMeterPSA::meterExecuteFunc(bool trace, P4::ReferenceMap* refMap) {
             "}\n"
             "\n"
             "static __always_inline\n"
+            "enum PSA_MeterColor_t meter_execute_color_aware(%meter_struct% *value, "
```

Comment:
I think it's more readable. Someone sometime will try to debug and will spend some time on thinking why there is a GREEN colour passed to this function ...

Reply:
https://datatracker.ietf.org/doc/html/rfc2698:
>  In the Color-Blind mode, the Meter assumes that the packet stream is uncolored.

So, maybe `uncolored` instead of `green` (they might equal value)?

Reply:
But we use enum from psa.h file. Adding there a `uncolored` value seems ugly